src/AWS.Logger.AspNetCore/NullExternalScopeProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using System;
+
+namespace AWS.Logger.AspNetCore
+{
+    /// <summary>
+    /// Scope provider that does nothing.
+    /// </summary>
+    internal class NullExternalScopeProvider : IExternalScopeProvider
+    {
+        private NullExternalScopeProvider()
+        {
+        }
+
+        /// <summary>
+        /// Returns a cached instance of <see cref="NullExternalScopeProvider"/>.
+        /// </summary>
+        public static IExternalScopeProvider Instance { get; } = new NullExternalScopeProvider();
+
+        /// <inheritdoc />
```

Comment:
At least in my experience, inheritdoc has flakey support. I don't think visual studio even shows hover docs for it.

Reply:
I would still rather use it at least in these methods that we don't provide any new implementation.